test/integration-test-main.sh

```diff
@@ -1047,7 +1047,7 @@ function test_truncate_cache() {
 function test_cache_file_stat() {
     describe "Test cache file stat ..."
 
-    dd if=/dev/urandom of="${BIG_FILE}" bs=${BIG_FILE_LENGTH} count=1
+    dd if=/dev/urandom of="${TEST_TEXT_FILE}" bs=$BIG_FILE_BLOCK_SIZE count=$BIG_FILE_COUNT
```

Comment:
This line is the reason why the test fails in Github Actions.
I think `BIG_FILE` is correct, not` TEST_TEXT_FILE`.

Reply:
Thanks!  I did not understand why this was failing.